src/controllers/projectCtrl.js

```diff
@@ -0,0 +1,92 @@
+const Project = require('../models/projectModel');
+
+exports.createProject = (req, res) => {
+  const newProject = new Project(req.body);
+  newProject
+    .save()
+    .then(() =>
+      res.status(201).json({ message: 'Project Created! :', newProject })
```

Comment:
Why do you put ':' at the end of the message ? And to normalize return data please write `data` instead of `newProject` 
```suggestion
      res.status(201).json({ message: 'Project Created!', data: newProject })
```